Mathlib/Algebra/Order/Floor/Semiring.lean

```diff
@@ -410,6 +410,13 @@ theorem floor_div_natCast (a : K) (n : ℕ) : ⌊a / n⌋₊ = ⌊a⌋₊ / n :=
   · exact lt_div_mul_add hn
   · exact cast_pos.2 hn
 
+theorem mul_floor_div_eq_floor {y : K} {b : ℕ} (hb : b ≠ 0) :
```

Comment:
Is the `Int.floor` version also true?

Reply:
It's true with a different RHS depending on the sign of `b`

Reply:
I would rather not have lemmas not about `Int.fdiv` mention `Int.fdiv`. Here having one version for each sign seems plenty reasonable